Neos.Utility.Files/Tests/Unit/FilesTest.php

```diff
@@ -41,6 +41,25 @@ public function tearDown()
     }
 
     /**
+     * @param string $target
+     * @param string $link
+     * @return boolean
+     * @throws \Exception
+     */
+    protected function trySymlink($target, $link)
+    {
+        try {
+            return symlink($target, $link);
+        } catch (\Exception $e) {
+            if (DIRECTORY_SEPARATOR !== '/') {
+                $this->markTestSkipped('Your windows installation does not allow symlinks. Make sure these tests are run with appropriate rights.');
```

Comment:
What about "Your Windows Installation does not allow the PHP process to create symlinks. Make sure that tests are executed with appropriate file permissions"?